yb-voyager/src/srcdb/yugabytedb.go

```diff
@@ -359,11 +361,54 @@ func (yb *YugabyteDB) FilterEmptyTables(tableList []*sqlname.SourceName) ([]*sql
 }
 
 func (yb *YugabyteDB) GetTableColumns(tableName *sqlname.SourceName) ([]string, []string, []string) {
-	return nil, nil, nil
+	var columns, dataTypes, dataTypesOwner []string
+	query := fmt.Sprintf(`SELECT a.attname AS column_name, t.typname AS data_type, rol.rolname AS data_type_owner 
```

Comment:
If it's the same query template as that of PG, try defining it as a variable ?

Reply:
Nope they are a little different

Reply:
Done